host/xtest/include/arm_ffa_user.h

```diff
@@ -0,0 +1,79 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
This file should probably go into `host/xtest/include/uapi/linux` until it's available by other means.

Reply:
Ack